### PR TITLE
added ability to add acquireKeyword to yaml

### PR DESCRIPTION
**Pull request by Lion-alt**
### What?
Add acquireKeyword to yaml
### Why?
To finish more sections.
### How?
Edited section-schema.json and a bit of ActionBuilder.

### CODE REVIEW

File: fabbwled-backend/section-schema.json [hunk 1]
```diff
@@ -23,6 +23,10 @@
               "description": "Text that should be shown to the user",
               "type": "string"
             },
+            "acquireKeyword": {
+              "description": "Acquires a keyword",
+              "type": "keyword"
```
Comment (Nilstrieb) [inline fabbwled-backend/section-schema.json:28]: type should be string

File: fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/service/ShardSystem.java [hunk 2]
```diff
@@ -1,11 +1,9 @@
 package ch.bbw.fabbwled.lands.service;
 
-import lombok.NoArgsConstructor;
+import lombok.Getter;
 
+@Getter
```
Comment (Nilstrieb) [inline fabbwled-backend/src/main/java/ch/bbw/fabbwled/lands/service/ShardSystem.java:5]: nice cleanup :)

**Review by Nilstrieb — CHANGES_REQUESTED** [review]

**Comment by Nilstrieb** [comment]
Also make sure to document this property in the Wiki.

**Review by Nilstrieb — APPROVED** [review]

**Review by Nilstrieb — APPROVED** [review]

**Review by dbabvwid — APPROVED** [review]

**Review by Rwael — APPROVED** [review]
Impressively modular architecture! 🧐

**Review by adrsch04 — APPROVED** [review]
looks good

**Review by MagischerWolf05 — APPROVED** [review]
👍

**Review by kimberlymo — APPROVED** [review]
lgtm 👍

**Review by lucasmillanm — APPROVED** [review]
Seems alright

**Pull request merged**